auth-web/tests/unit/components/AffiliatedEntityTable.spec.ts

```diff
@@ -40,54 +42,75 @@ const businessModule = {
   }
 }
 
+const moreBusinessModule = {
+  namespaced: true,
+  state: { businesses: moreBusinesses },
+  action: {
+    addBusiness: jest.fn(),
+    updateBusinessName: jest.fn(),
+    updateFolioNumber: jest.fn()
+  }
+}
+
 sessionStorage.setItem('AUTH_API_CONFIG', JSON.stringify({
   AUTH_API_URL: 'https://localhost:8080/api/v1/11',
   PAY_API_URL: 'https://pay-api-dev.apps.silver.devops.gov.bc.ca/api/v1'
 }))
 
-const store = new Vuex.Store({
+const oldStore = new Vuex.Store({
   strict: false,
   modules: {
     business: businessModule
   }
 })
 
+const newStore = new Vuex.Store({
+  strict: false,
+  modules: {
+    business: moreBusinessModule
+  }
+})
+
 const vuetify = new Vuetify({})
 
 describe('AffiliatedEntityTable.vue', () => {
   setupIntersectionObserverMock()
   let wrapper: Wrapper<any>
+  let localVue: VueConstructor<any>
 
   const headers = getAffiliationTableHeaders(['Number', 'Type', 'Status'])
 
   beforeEach(async () => {
-    const localVue = createLocalVue()
-    wrapper = mount(AffiliatedEntityTable, {
-      store,
-      localVue,
-      vuetify,
-      propsData: { selectedColumns: ['Number', 'Type', 'Status'] },
-      mocks: { $t: () => '' }
-    })
+    localVue = createLocalVue()
   })
 
   afterEach(() => {
     wrapper.destroy()
+    sessionStorage.clear()
 
     jest.resetModules()
     jest.clearAllMocks()
   })
 
-  it('Renders affiliated entity table', async () => {
-    // verify table header
-    expect(wrapper.find('.table-header').text()).toBe('My List (7)')
+  it('Renders affiliated entity table with old actions menu', async () => {
+    sessionStorage.__STORE__[SessionStorageKeys.LaunchDarklyFlags] = JSON.stringify({ 'enable-new-actions-menu': false })
```

Comment:
What do you think about mocking the sessionStorage call instead? I'm OK either way.

Reply:
This is the way other tests do it, so leave it this way. Resolving.